src/page-multi-facility/types.tsx

```diff
@@ -22,8 +22,9 @@ export type Scenario = {
   baseline: boolean;
   dataSharing: boolean;
   dailyReports: boolean;
+  promoStatuses?: PromoStatuses | undefined;
   description: string;
-  roles: Map<string, string>;
+  roles: object;
```

Comment:
Was there a reason this changed from `Map<string, string>` to `object`?

Reply:
Oops, accidentally left that in from when I was trying to add a test and was seeing a type error when I tried to create the field as a Map type. I can change it back